research/src/pages/select/select.proposal.mdx

```diff
@@ -116,11 +116,55 @@ children:
+| `click`          | Toggles the open state on the host | [open](#open-state) state                                                |
+| `click`          | Toggles aria-expanded              | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+| `pointerup`      | Toggles the open state on the host | [open](#open-state) state                                                |
+| `pointerup`      | Toggles aria-expanded              | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+| `keydown(space)` | Toggles the open state on the host | [open](#open-state) state                                                |
+| `keydown(space)` | Toggles aria-expanded              | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+| `keydown(enter)` | Toggles the open state on the host | [open](#open-state) state                                                |
+| `keydown(enter)` | Toggles aria-expanded              | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
+
+### part listbox
+
+| Event                 | Behavior                                                                                                               | Impacts                                                                  |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------ |
+| `keydown(down key)`\* | Moves focus to the next `<option>` in the listbox                                                                      | focus                                                                    |
+| `keydown(up key)`\*   | Moves focus to the previous `<option>` in the listbox                                                                  | focus                                                                    |
+| `keydown(enter)`      | Changes the `selected` state of the current `<option>` and updates the `<select>` value property                       | [selected](#selected) prop <br/> [value](#value) prop                    |
+| `keydown(space)`      | Changes the `selected` state of the current `<option>` and updates the `<select>` value property                       | [selected](#selected) prop <br/> [value](#value) prop                    |
+| `keydown(enter)`      | If the `<select>` does **not** have the `multiple` attribute then toggle the state of `open` on the `<select>`         | [open](#open-state) state                                                |
+| `keydown(enter)`      | If the `<select>` does **not** have the `multiple` attribute then toggle the aria-expanded attribute on the `<select>` | [aria-expanded](#https://www.w3.org/TR/wai-aria-1.1/#aria-expanded) attr |
```

Comment:
I'm thinking that `aria-expanded` should be set on the `button` part, not the `<select>` itself.  I think that should generally be the case with these `aria-*` semantics.  It's the parts in the shadow, not the host element, that are getting rendered, so they are the ones that need the ARIA fixup.